xml/tuning-intro.xml

```diff
@@ -0,0 +1,366 @@
+ <title>Introduction to Tuning &productname; Clusters </title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
+   <dm:translation>yes</dm:translation>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+   Tuning a distributed cluster isa foray into the use of the
+   Scientific Method backed with iterative testing. By taking a wholistic
+   look at the cluster and then delving into all the components, it is
+   possible to achieve dramatic improvements. Over the
+   course of the work that contributed to the authoring of this guide,
+   the authors have seen performance more than double in some specific cases.
+ </para>
+ <para>
+   This guide is intended to assist the reader in understanding the what
+   and how of tuning a &productname; cluster. There are topics
+   that are beyond the scope of this guide, and it is expected that
+   there are further tweaks that may be performed to an individual cluster
+   in order to achieve optimum performance in a particular end user environment.
+ </para>
+ <para>
+   This reference guide is targeted at architects and administrators who need
+   to tune their &productname; cluster for optimal performance.
+   Familiarity with Linux and &ceph; are assumed.
+ </para>
+ <sect1 xml:id="tuning-philosophy">
+   <title>Philosophy of Tuning</title>
+   <para>
+     Tuning requires looking at the entire system being tuned and approaching
+     the process with scientific rigour. An exhaustive approach requires
+     taking an initial baseline and altering a single variable at a time,
+     measuring the result, and then reverting it back to default while
+     moving on to the next tuning parameter. At the end of that process,
+     it is then possible to examine the results, whether it be increased
+     throughput, reduced latency, reduced CPU consumption, and then
+     decide which are likely candidates for combining for additive benefit.
+     This second phase should also be iterated through in the same fashion
+     as the first. This general process would be continued until all possible
+     combinations were tried and the optimal settings discovered.
+   </para>
+   <para>
+     Unfortunately, few have the time to perform such an exhaustive
+     effort. This being reality, it is possible to utilize some knowledge
+     and begin an iterative process of combining and measuring well-known
+     candidates for performance improvements and measuring the resulting
+     changes. That is the process followed during the research that produced
+     this work.
+   </para>
+ </sect1>
+ <sect1 xml:id="tuning-process">
+   <title>The Process</title>
+   <para>
+     A proper process is required for effective tuning to occur.
+     The tenants of this process are:
```

Comment:
I think that they mean "tenets" here. It is safe to assume that nobody rents space and lives in the process.

Reply:
Shame, really :joy: